content/en/blog/2022/OpenTelemetry-In-Practice-August.md

```diff
@@ -0,0 +1,55 @@
+---
+title: OpenTelemetry in Practice - Kubernetes & the Collector
+linkTitle: OTel in Practice
+date: 2022-08-11
+author: Rynn Mancuso
+draft: true
+# TODO: remove this line once your post is ready to be published
+# TODO: run the following command before submitting
+#   npx prettier --write content/en/blog/2022/OpenTelemetry-In-Practice-August.md
+---
+
+## About the OpenTelemetry in Practice Series
+
+Welcome to the OpenTelemetry in Practice series! This is a new experiment by
```

Comment:
Is there a canonical link we can put here for `OpenTelemetry in Practice` for folks?

Reply:
Maybe somewhere in the CNCF slack or something

Reply:
Currently, this blog post is the only thing available about the event and the event is intended to be the first event in a series. I followed the same process Sharr did with the end-user lean coffee - create [blog post](https://opentelemetry.io/blog/2022/new-end-user-resources/) to announce event, get event added to calendar.

What would you suggest that I use as a canonical link? Is there some other material I should create? Let me know and I'll open a PR.

Reply:
Gotcha, makes sense. No real suggestions on my end, so it sounds good to me!